src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql

```diff
@@ -0,0 +1,14 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:moto_masking_account_update_permission
+INSERT INTO permissions(id, name, description) VALUES (49, 'moto-mask-input:update', 'Update Moto Masking Account');
```

Comment:
The description should be something like “Update MOTO mask input settings”.

Reply:
ok will make that change